src/app/pages/jails/jail-add/jail-add.component.ts

```diff
@@ -118,538 +130,661 @@ export class JailAddComponent implements OnInit {
-      tooltip: T('Mount a <a\
- href="https://www.freebsd.org/cgi/man.cgi?query=devfs&sektion=5&manpath=FreeBSD+11.1-RELEASE+and+Ports"\
- target="_blank">devfs(5)</a> filesystem on the chrooted <ins>/dev</ins>\
- directory and apply the ruleset in the <b>devfs_ruleset</b> parameter\
- to restrict the devices visible inside the jail.'),
-    }, {
+      placeholder: T('mount.devfs'),
+      tooltip: T('Mount a <a \
+                  href="https://www.freebsd.org/cgi/man.cgi?query=devfs&sektion=5&manpath=FreeBSD+11.1-RELEASE+and+Ports"\
+                  target="_blank">devfs(5)</a> filesystem on the \
+                  <i>chrooted /dev directory</i> and apply the ruleset \
+                  in the <b>devfs_ruleset</b> parameter to restrict \
+                  the devices visible inside the jail.'),
+    },
+    {
       type: 'checkbox',
       name: 'mount_fdescfs',
-      placeholder: T('mount_fdescfs'),
-      tooltip: T('Mount an <a\
- href="https://www.freebsd.org/cgi/man.cgi?query=fdescfs&sektion=5&manpath=FreeBSD+11.1-RELEASE+and+Ports"\
- target="_blank">fdescfs(5)</a> filesystem in the jail <ins>/dev/fd</ins>\
- directory.'),
-    }, {
+      placeholder: T('mount.fdescfs'),
+      tooltip: T('Mount an <a \
+                  href="https://www.freebsd.org/cgi/man.cgi?query=fdescfs&sektion=5&manpath=FreeBSD+11.1-RELEASE+and+Ports"\
+                  target="_blank">fdescfs(5)</a> filesystem in the \
+                  jail <i>/dev/fd</i> directory.'),
+    },
+    {
       //"enforce_statfs": ("0", "1", "2"),
       type: 'select',
       name: 'enforce_statfs',
       placeholder: T('enforce_statfs'),
-      tooltip: T('Determine which information processes in a jail are able\
- to obtain about mount points.  The behavior of multiple syscalls is\
- affected. <a\
- href="https://www.freebsd.org/cgi/man.cgi?query=statfs&manpath=FreeBSD+11.1-RELEASE+and+Ports"\
- target="_blank">statfs(2)</a>, <a\
- href="https://www.freebsd.org/cgi/man.cgi?query=statfs&manpath=FreeBSD+11.1-RELEASE+and+Ports"\
- target="_blank">fstatfs(2)</a>, <a\
- href="https://www.freebsd.org/cgi/man.cgi?query=getfsstat&manpath=FreeBSD+11.1-RELEASE+and+Ports"\
- target="_blank">getfsstat(2)</a>, and <a\
- href="https://www.freebsd.org/cgi/man.cgi?query=fhstatfs&manpath=FreeBSD+11.1-RELEASE+and+Ports"\
- target="_blank">fhstatfs(2)</a> and other similar compatibility syscalls.\
- <br>When set to <i>0</i>, all mount points are available without any\
- restrictions.  When set to <i>1</i>, only mount points below the jail\
- chroot directory are visible. When set to <i>2</i>, the syscalls above\
- can operate only on a mountpoint where the jail chroot directory is\
- located.'),
-      options: [{
-        label: 'O',
-        value: '0',
-      }, {
-        label: '1',
-        value: '1',
-      }, {
-        label: '2',
-        value: '2',
-      }]
-    }, {
+      tooltip: T('Determine which information processes in a jail are \
```

Comment:
s/information processes/information the processes/ to separate the two words.

Reply:
fixed